### PR TITLE
Implemented refund

### PR DESCRIPTION
**Pull request by g4s8**
#2007 - refund command for project

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#2008](https://github.com/zerocracy/farm/issues/2008) is now in scope, role is `REV`

File: src/main/groovy/com/zerocracy/stk/pm/refund.groovy [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      new Par(
+        farm,
+        'We can\'t refund for %s because you don\'t have enough cash',
+        'or it was estimated and locked.',
+        'cash/estimated/available: %s/%s/%s',
+      ).say(amount, cash, locked, available)
+    )
+  }
+  String login = claim.author()
+  ledger.add(
```
Comment (g4s8) [inline src/main/groovy/com/zerocracy/stk/pm/refund.groovy]: @yegor256 can you please check that it's correct refund transaction? I'm not sure about ct, ctx, dt and dtx values here.

Comment (yegor256) [reply src/main/groovy/com/zerocracy/stk/pm/refund.groovy]: @g4s8 looks correct

**Comment by yegor256** [comment]
@g4s8 looks good to me, let's merge and release, thanks

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor stop

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#2008` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#2008](https://github.com/zerocracy/farm/issues/2008) is now out of scope

**Pull request merged**